extended_report.go

```diff
@@ -646,7 +646,8 @@ func (x *ExtendedReport) Unmarshal(b []byte) error {
 
 // DestinationSSRC returns an array of SSRC values that this packet refers to.
 func (x *ExtendedReport) DestinationSSRC() []uint32 {
-	ssrc := make([]uint32, 0)
+	ssrc := make([]uint32, 0, len(x.Reports)+1)
+	ssrc = append(ssrc, x.SenderSSRC)
```

Comment:
That function name/meaning is specifically for destination. Guess, for XR case, it is okay to include sender also, but wondering if there is a cleaner option.

Reply:
It is not a naming issue, the DestinationSSRC used by pion to dispatch rtcp packets, the packet will be dropped if it doesn't include the SenderSSRC and report block doesn't contain a ssrc field.

Reply:
Got it. Thank you @cnderrauber